lib/experimental/rest-api.php

```diff
@@ -100,3 +100,12 @@ function gutenberg_auto_draft_get_sample_permalink( $permalink, $id, $title, $na
 	return $permalink;
 }
 add_filter( 'get_sample_permalink', 'gutenberg_auto_draft_get_sample_permalink', 10, 5 );
+
+add_action( 'wp_ajax_gutenberg_fetch_media', 'gutenberg_fetch_media' );
+
+function gutenberg_fetch_media() {
```

Comment:
We should do a caps check for `edit_posts` and friends, for example what we do in the patterns directory controller.

Reply:
just media related caps?

Reply:
Ah, yeah that's probably better I think.

Reply:
I figured out some weird trick where if I append `?` to the image URL to skip cache, the CORS errors disappear. So I removed this server side code. I didn't find any images that fail now.